### PR TITLE
add development instructions

### PR DESCRIPTION
**Pull request by tjulien**

### CODE REVIEW

**Comment by tjulien** [comment]
https://app.clubhouse.io/radarlabs/story/6840/document-ios-and-android-development-process-in-the-readmes

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Documentation on building the SDK
+To do development on the SDK source, use the following commands:
+
+```
+brew install cocoapods # if you do not have cocoapods installed
+pod install
```
Comment (david-goodfellow) [inline DEVELOPMENT.md:6]: I get the following in the project root folder on this line:
```
❯ pod install
[!] No `Podfile' found in the project directory.
```

Comment (david-goodfellow) [reply DEVELOPMENT.md:6]: actually this step was unneeded for me. Only step I had to do was the `xcpretty` was not installed so I had to run `sudo gem install xcpretty`. Was fairly straightforward to debug this so don't know if we need to explicitly list out the dependencies

**Comment by nickpatrick** [comment]
Yeah, you shouldn't need to do a `pod install` to build the SDK. But we do a `pod lint` to make sure the `Podfile` is properly formatted as part of `make dist`.

There is a `make bootstrap` command that will run `bootstrap.sh` to install all dependencies, including `xcpretty` and `cocoapods`.

**Comment by nickpatrick** [comment]
@tjulien Closing this and will rewrite in the 3.2 branch

**Pull request closed without merging**